edalize/templates/libero/libero-project.tcl.j2

```diff
@@ -0,0 +1,98 @@
+# This file is generated by Edalize.
+# Microsemi Tcl Script
+# Libero
+
+# Create a new project with device parameters
+new_project -location {{op}}{{prj_root}}{{cl}} -name {{name}} -project_description {} -block_mode 0 -standalone_peripheral_initialization 0 -instantiate_in_smartdesign 1 -ondemand_build_dh 1 -hdl {{op}}{{tool_options.hdl}}{{cl}} -family {{op}}{{tool_options.family}}{{cl}} -die {{op}}{{tool_options.die}}{{cl}} -package {{op}}{{tool_options.package}}{{cl}} {% if tool_options.speed -%} -speed {{op}}{{tool_options.speed}}{{cl}}{{sp}}{%- endif %}{% if tool_options.dievoltage -%} -die_voltage {{op}}{{tool_options.dievoltage}}{{cl}}{{sp}}{%- endif %}{% if tool_options.range -%} -part_range {{op}}{{tool_options.range}}{{cl}}{{sp}}{%- endif %}{% if tool_options.defiostd -%} -adv_options {IO_DEFT_STD:{{tool_options.defiostd}}{{cl}}{{sp}}{%- endif %}-adv_options {RESTRICTPROBEPINS:1} -adv_options {RESTRICTSPIPINS:0} -adv_options {SYSTEM_CONTROLLER_SUSPEND_MODE:0} -adv_options {TEMPR:{{tool_options.range}}{{cl}} -adv_options {VCCI_1.2_VOLTR:EXT} -adv_options {VCCI_1.5_VOLTR:EXT} -adv_options {VCCI_1.8_VOLTR:EXT} -adv_options {VCCI_2.5_VOLTR:EXT} -adv_options {VCCI_3.3_VOLTR:EXT} -adv_options {VOLTR:{{tool_options.range}}{{cl}}
```

Comment:
Drop all the `adv_options`. I believe most of those can be set in otherwise if users need them. Also drop
-block_mode 
-standalone_peripheral_initialization
-instantiate_in_smartdesign
-ondemand_build_dh

In fact, I diffed to project files and most of these options doesn't seem to do anything at all

Reply:
Done. I just kept default IO std since it's already a user parameter.